middlewares/logger.js

```diff
@@ -0,0 +1,31 @@
+import { createLogger, format, transports } from 'winston';
+const { combine, timestamp } = format;
+
+const logger = createLogger({
+    level: 'info',
+    format: combine(
+        timestamp({
+            format: 'YYYY-MM-DD HH:mm:ss'
+        }),
+        format.simple()
+    ),
+
+    transports: [
+        new transports.Console(),
+        new transports.File({ filename: 'error.log', level: 'error' }),
+        new transports.File({ filename: 'combined.log' })
+    ]
+});
+
+
+
+exports.logInfo = function(req, res, next){
```

Comment:
Great middleware/approach for looging :+1: . Enhancement: just return anonymous function.
```suggestion
module.exports = function(req, res, next){
```